src/commands/webViewCommand.ts

```diff
@@ -98,6 +98,13 @@ export class WebViewCommand {
 
         // Start to load the changes tab, gets called everytime a new sast details webview is opened
         await this.loadAsyncTabsContent(result);
+
+        vscode.window.onDidChangeActiveTextEditor(async (editor) => {
+          if (!editor) {
```

Comment:
What is editor?

Reply:
The editor parameter represents the active text editor in Visual Studio Code, so when we return to plugin tab means exit from edit file so we need to load the data again.